src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -601,7 +601,7 @@ public CacheWriter(ResolvePackageAssets task, Stream stream = null)
 
                 _task = task;
                 _lockFile = new LockFileCache(task).GetLockFile(task.ProjectAssetsFile);
-                _packageResolver = NuGetPackageResolver.CreateResolver(_lockFile, _task.ProjectPath);
+                _packageResolver = NuGetPackageResolver.CreateResolver(_lockFile);
```

Comment:
Can we remove the ProjectPath parameter from some of these other tasks that were passing it to `CreateResolver`, or are they still using it for other things?

Reply:
Some are still using it. Do you think it's worth scrubbing? I can do it if so.

Reply:
There was only one more use of ProjectPath that was no longer needed.  Fixed. The other tasks use it for various other purposes.